omod/src/main/java/org/openmrs/module/htmlformentryui/page/controller/htmlform/ViewEncounterWithHtmlFormPageController.java

```diff
@@ -52,8 +65,13 @@ public void get(@RequestParam("encounter") Encounter encounter,
         model.addAttribute("showPatientHeader", showPatientHeader);
 
         HtmlForm htmlForm = htmlFormEntryService.getHtmlFormByForm(encounter.getForm());
+        
         if (htmlForm == null) {
-            throw new IllegalArgumentException("encounter.form is not an HTML Form: " + encounter.getForm());
+             message = messageSourceService.getMessage("encounter.form is not an HTML Form" +encounter.getForm());
+            log.warn("Active drugs are cannot be editted");
+            model.addAttribute("htmlForm", htmlForm);
```

Comment:
@sherrif10  already here the form be null , why then add a null to the model ??

Reply:
Thanks @mozzy11 , Wanted to return a model rathan null, let me re-correct it